pkg/exemplars/proxy.go

```diff
@@ -80,10 +81,11 @@ func (s *Proxy) Exemplars(req *exemplarspb.ExemplarsRequest, srv exemplarspb.Exe
 	)
 
 	for _, st := range s.exemplars() {
-		query := ""
+		var queryParts []string
```

Comment:
We can probably reuse `queryParts` slice instead of initializing it to nil every time.